iOS/Flows/Conversation/ConversationViewControllerDataSource.swift

```diff
@@ -66,7 +66,7 @@ class ConversationCollectionViewDataSource: CollectionViewDataSource<Conversatio
 
     /// Updates the datasource with the passed in array of message changes.
     func update(with changes: [ListChange<Message>],
-                conversationController: ConversationController,
+                conversationController: ConversationControllerType,
```

Comment:
A common interface for message and conversation controllers.

Reply:
yep. This makes total sense.